backend/pick-git/src/main/java/com/woowacourse/pickgit/post/domain/Post.java

```diff
@@ -44,4 +47,29 @@
 
     protected Post() {
     }
+
+    public Post(Images images, PostContent content, Likes likes, Comments comments, User user) {
+        this.images = images;
+        this.content = content;
+        this.likes = likes;
+        this.comments = comments;
+        this.user = user;
+    }
+
+    public void addTags(List<Tag> tags) {
+        List<Tag> existingTags = getTags();
+        for (Tag tag : tags) {
+            if (existingTags.contains(tag)) {
+                throw new IllegalArgumentException("중복되는 태그를 추가할 수 없습니다.");
+            }
+            PostTag postTag = new PostTag(this, tag);
+            postTags.add(postTag);
+        }
+    }
+
+    public List<Tag> getTags() {
+        return postTags.stream()
+            .map(PostTag::getTag)
+            .collect(Collectors.toList());
```

Comment:
```suggestion
            .collect(toList());
```

static import 해주세요 !